compiler/circle-opselector/pass/Function1.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "Function1.h"
```

Comment:
Please add an empty line above this line

Reply:
Just curious, 
(1) why is the file name `Function1` ? It woud be better to specifiy it;s purpose as a name.
(2) what is `pass` mean? is this for testing purpose?

Reply:
We plan to..
- Change `function1` to specific function name
- Remove `singlePass`

While we were drafting the `OpSelector`, we were certain that `pass` and `function1` are for '***selecting node***'. But, we were not certain about '***what specific functions would be needed***'.
So, we came to the idea, 'How about creating a **chained function queue** that performs the required functions sequentially, and **add necessary functions later?**'
We are planning to name this virtual class as `SinglePass` which can be overidden at `function1, 2, 3, ...`
But now, it seems there is no necessary chained function queue, so we will **remove** `singlePass` and **rename** `function1` to it's **specific purpose.**